controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -119,8 +134,11 @@ def main() -> None:
 
     remove_unused_robots(supervisor)
 
-    with record_animation(supervisor, REPO_ROOT / 'recordings' / recording_path()):
-        run_match(supervisor)
+    recording_path = get_recording_path()
+
+    with record_animation(supervisor, REPO_ROOT / 'recordings' / '{}.html'.format(recording_path)):
+        with record_video(supervisor, REPO_ROOT / 'recordings' / '{}.mp4'.format(recording_path)):
```

Comment:
It looks like we've lost a layer of directories here? Bear in mind that the animation outputs several files, so it's advantageous to have a subdirectory per run.

I would also suggest that we pull the construction of the path upwards to reduce the complexity here. I think we want something a bit like this:
```suggestion
    recording_name = get_recording_path()
    recording_path = REPO_ROOT / 'recordings' / recording_name

    with record_animation(supervisor, recording_path / '{}.html'.format(recording_name)):
        with record_video(supervisor, recording_path / '{}.mp4'.format(recording_name)):
```

Reply:
cb818d4